Automattic/jetpack
# Dialogue: edit participant from canvas

This PR implements a new component used to edit the participants in a conversation structure.

https://user-images.githubusercontent.com/77539/106919456-ec6dd980-66e8-11eb-96a3-92f5f520cc12.mov


#### Changes proposed in this Pull Request:

* Dialogue: edit participant from canvas

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


NO

#### Testing instructions:






* Create a new conversation block. Check you can get it by `conversation`, `transcription`, `dialogue`, `dialog`, and `speaker` keywords.
* When you start a new component from scratch, there is not any participant defined in the conversation context, thus you need to start to type the participant name in some of the dialogue blocks.
* The new participant will be added once press `ENTER` key or when on focus outside
* When editing a new Dialogue participant, you should get the participant suggestions.
* To add a new participant, you need to clean the participant content before starting to type. Otherwise, it will edit the current participant and all sibling Dialogue blocks with the same participant.

D56506-code

#### Proposed changelog entry for your changes:


*
